build.sbt

```diff
@@ -111,14 +111,12 @@ lazy val rainierCore = project.
   dependsOn(rainierCompute, rainierSampler).
   settings(commonSettings)
 
-lazy val rainierPlot = project.
-  in(file("rainier-plot")).
-  settings(name := "rainier-plot").
-  dependsOn(rainierCore).
+lazy val rainierNotebook = project.
+  in(file("rainier-notebook")).
+  settings(name := "rainier-notebook").
   settings(commonSettings).
```

Comment:
I only needed to delete one method to make rainier-notebook a totally standalone package.